linebot/models/flex_message.py

```diff
@@ -276,6 +356,11 @@ def __init__(self, action=None, flex=None, margin=None, height=None, style=None,
         :param str color: Character color when the style property is link.
             Background color when the style property is primary or secondary.
             Use a hexadecimal color code
+        :param str position: Reference position for placing this box
```

Comment:
[nits]
The order of parameters in docstring is not the same as that in the method definition

Reply:
fixed